ui/components/MesheryMeshInterface/PatternService/helper.js

```diff
@@ -221,6 +221,9 @@ export const calculateGrid = element => {
     grid.lg = 12;
   }
 
+  // if fields have custom annotations to grid them differently
+  grid.lg = element.content.props.schema["x-rjsf-style"] || grid.lg;
```

Comment:
Good work @harkiratsm, since this annotation is just meant for grid-area, would you rename it to `x-rjsf-grid-area`?

Reply:
Would you please add a screenshot after your changes.

Reply:
![image](https://github.com/meshery/meshery/assets/71957674/08587388-6147-4426-a366-e9cbdbc674cb)